docs/modules/knowledge-graph/pages/knowledge-graph.adoc

```diff
@@ -0,0 +1,101 @@
+= Knowledge Graph
+
+Knowledge Graph is a RAGStack library that provides graph-based representation and retrieval of information. It is designed to store and retrieve information in a way that is more efficient and accurate than vector-based similarity search over Document chunks.
+
+See the xref:examples:knowledge-graph.adoc[Knowledge graph example code] to get started using Knowledge Graph RAG.
+
+== The `ragstack-ai-knowledge-graph` library
+
+The `ragstack-ai-knowledge-graph` library contains functions for the extraction and traversal of knowledge graphs.
+
+To install the library, run:
+
+[source,bash]
+====
+pip install ragstack-ai-knowledge-graph
+====
+
+== How is Knowledge Graph different from RAG?
```

Comment:
Maybe worth starting out with: It isn't. It is a different representation for storing information, but can still be used to retrieve and augment generated results.

Specifically, vector is a way to do RAG, but knowledge graphs are too.

Reply:
Added, thanks for clarification